test/e2e/scheduling/predicates.go

```diff
@@ -936,7 +940,9 @@ func WaitForSchedulerAfterAction(f *framework.Framework, action Action, ns, podN
 	}
 	success, err := observeEventAfterAction(f.ClientSet, f.Namespace.Name, predicate, action)
 	framework.ExpectNoError(err)
-	framework.ExpectEqual(success, true)
+	if !success {
+		framework.Failf("no event matching the predicate was emitted from the system after performing the supplied action for pod %v", podName)
```

Comment:
It would be helpful to have a different message for Failure vs Success.

Reply:
Done!

Reply:
"predicate" is an implementation detail of this function. That's not what we want to log.

We want to know whether there was a Success event (when `expectSuccess=true`) or a Failure event (when `expectSuccess=false`).

Reply:
@Ahmed-Aghadi I think this is still outstanding. Can you address this?

Reply:
@alculquicondor  so should I replace this with : 
`framework.Logf("event was successful after performing the supplied action for pod %v", podName)`
and 
`framework.Logf("event  was failed after performing the supplied action for pod %v", podName)`

Reply:
There shouldn't be `Logf`, just `Failf` if we were expecting Success but got Failed, or we were expecting Failed but got Success.

Reply:
@alculquicondor Done!